src/data/relicansData.js

```diff
@@ -356,9 +356,11 @@ export const teamMembers = [
     avatar: rachaelAvatar,
     bio: (
       <p>
-        Rachael's been building software and web apps since 2012. Now she builds
-        her open-source vue/rails app live on Twitch. Enough people thought that
-        was cool, so she's been invited here.
+        Rachael wants to share the joy and beauty of programming
+        through programming games and regular open-source streams on Twitch.
+        She's been a professional dev since 2012, 3x Team Lead, and manages an
```

Comment:
```suggestion
She's been a professional dev since 2012, 3x Team Lead, and manages 
an open-source repo for playing games of Monster of the Week. Currently, 
she works as a Developer Relations Engineer for New Relic.
```